src/auth/authenticationManager.ts

```diff
@@ -143,12 +68,15 @@ export function useIsIdTokenExpired() {
       );
 
       navigate(bookingListPath, { state: pathname });
+    } else if (refreshTokenTriggerResult.isError) {
+      // navigate(errorPath);
```

Comment:
Let's go over the application and remove any unneeded comments

Reply:
removed